ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml

```diff
@@ -0,0 +1,58 @@
+base_images:
```

Comment:
Also remove config for any release-4.7+ release branches - future-looking branches config is also automatically managed.

Reply:
Excellent, the 4.8 release files have also been removed :+1:

Reply:
They still seem to be here though ;)

Reply:
4.7+! Ah ha! Shoot thanks for the help Petr, I just took out the 4.8 files, thanks for that.

Reply:
One more deletion please :)

`ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml`